src/services/message.service.ts

```diff
@@ -31,6 +31,46 @@ export class MessageService {
     this._sendConstructedReport(report, { files: message.attachments.map((e) => e.url) });
   }
 
+  async sendMessage(
+    message: IMessage,
+    content: string,
+    reply: boolean,
+    argHeader?: string,
+    argFooter?: string,
+    argBreakChar?: string
+  ): Promise<Boolean> {
+    const header = argHeader ? argHeader + '\n' : '';
+    const footer = argFooter ? '\n' + argFooter : '';
+    const breakChar = argBreakChar || '';
+
+    const replyLength = reply ? ('<@' + message.member?.id + '>,  ').length : 0;
+    const miscLength = header.length + footer.length;
+
+    let messagesToSend: string[] = [header + content + footer];
+
+    if (content.length + miscLength + replyLength > Constants.MaxMessageLength) {
+      if (!breakChar) {
+        return this._failedToSend(message, `Message was over Discords character cap.`);
+      }
+
+      messagesToSend = this._splitMessage(content, breakChar, replyLength + miscLength);
+      if (!this._testMessages(messagesToSend, content.length, replyLength)) {
+        return this._failedToSend(message, `Failed to split the message up within the rule set.`);
+      }
+      messagesToSend = messagesToSend.map((message) => header + message + footer);
+    }
+
+    if (reply) {
+      message.reply(messagesToSend.shift() || '');
```

Comment:
if `reply` is true, shouldn't we reply to the user with all chunks of the message?

Reply:
we can, but I think it looks better not having your tag on all of them.  This is what it looks like when you do it the current way for reference.
![image](https://user-images.githubusercontent.com/10034057/113013260-9007c300-9149-11eb-8a5b-d7cd414a91c1.png)